docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,178 @@
 ---
-title: Smart Optimistic Rollups
-authors: 'Nomadic Labs, TriliTech'
+title: Smart Rollups
+authors: 'Nomadic Labs, TriliTech, Tim McMackin'
 last_update:
-  date: 30 June 2023
+  date: 4 Jan 2024
 ---
 
-Rollups play a crucial part in providing next-generation scaling on Tezos. This page gives a technical introduction to Smart Rollups, their optimistic nature, and an intro to developing your own WASM kernel.
+import LucidDiagram from '@site/src/components/LucidDiagram';
 
-## Examples
+Smart Rollups play a crucial part in providing high scalability on Tezos.
+They handle transactions and logic in a separate environment that doesn't need to follow all of the same rules as the main Tezos network.
+For this reason, they are not subject to the same block delays and data space constraints as the main Tezos network.
 
-For examples of Smart Rollups, see this repository: https://gitlab.com/tezos/kernel-gallery.
+The transactions and logic that Smart Rollups run is called _layer 2_ to differentiate it from the main network, which is called _layer 1_.
+
+The compiled code of a Smart Rollup must be provided publicly, so that anyone can run a Smart Rollup node to execute that code and verify that other nodes are running it correctly, just like anyone can run nodes, bakers, and accusers on layer 1.
+This code, called the _kernel_, must behave in a deterministic manner and according to a given semantics, which guarantees that results are reproducible by any rollup node with the same kernel.
```

Comment:
Deterministic execution is ensured by the reference PVM, so it is virtually impossible to write a kernel that behaves non-deterministically

Reply:
Changed to:
> This code, called the _kernel_, runs in a deterministic manner...